cleaner/cleaner.go

```diff
@@ -0,0 +1,35 @@
+/*
+Copyright 2022 Red Hat, Inc. and/or its affiliates.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cleaner
+
+import (
+	"context"
+	"github.com/docker/docker/client"
+)
+
+type RegistryCleaner interface {
+	RemoveImagesUntagged()
```

Comment:
I think these methods should return something, a bool flag or the info about the removed container together a error.

Reply:
the bool and the error are ok, the info it's a little bit hard because Docker and podman provides different struct for that info